saltproc/tests/test_depcode.py

```diff
@@ -84,11 +95,20 @@ def test_get_nuc_name():
 
 def test_read_depcode_info():
     serpent.read_depcode_info()
-    assert serpent.sim_info['serpent_version'] == 'Serpent 2.1.31'
+    assert serpent.sim_info['depcode_name'] == 'Serpent'
+    assert serpent.sim_info['depcode_version'] == '2.1.31'
```

Comment:
what if user updates serpent? Will saltproc not work?

Reply:
The saltproc unit test suite has some dummy files it uses to check functions that do file IO. This particular test is not checking that the serpent version is 2.1.31, but that that `sim_info['depcode_version']` matches the value stored in the summary output file.